tests/vppinputoutput.h

```diff
@@ -280,6 +281,7 @@ class VppOutput
                                        const char* codecName = NULL);
     bool getFormat(uint32_t& fourcc, int& width, int& height);
     virtual bool output(const SharedPtr<VideoFrame>& frame) = 0;
+    virtual void printLatency() = 0;
```

Comment:
this not need be pure virtual, in you declear it as `virtual void printLatency(){};` and remove "void printLatency(){};" in VppOutputFile